app/assets/javascripts/modules/accordion-with-descriptions.js

```diff
@@ -45,6 +47,24 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
       bindToggleForSubsections();
       bindToggleOpenCloseAllButton();
 
+      // When navigating back in browser history to the accordion, the browser will try to be "clever" and return
```

Comment:
An example on GOV.UK where cmd+clicking acts as a toggle: https://www.gov.uk/guidance/the-highway-code/rules-for-pedestrians-1-to-35